README.md

```diff
@@ -4,42 +4,52 @@ This is a skeleton for confidential Oasis dApps in Python.
 
 ## Prerequisites
 
-This project was tested on python 3.10, but should work with most python3 versions. 
+This project was tested on python 3.10, but should work with most 
+python3 versions. 
 Use pyenv to handle multiple python installations.
 
-1. Initialize an environment using preferred environment manager (venv, pipx...) ```python3 -m venv my_env``` 
-and install the python sapphire wrapper. 
-The library is currently **NOT** included in the PyPI repository so make sure to only install the local .whl version.
+## Installation
 
-2. Make sure to follow the [instructions](https://github.com/oasisprotocol/sapphire-paratime/tree/main/clients/py "Py client wrapper") and build the .whl file correctly. 
-In the Py client wrapper dist/ folder, find the .whl file and install 
-it by running ```pip install <path_to_whl_file>```.
+1. Initialize an environment using preferred environment manager 
+(venv, pipx...) ```python3 -m venv my_env```. 
+There are two ways to install the wrapper client:
 
-3. Install the packages in requirements.txt ```pip install -r requirements.txt```
+    - Build the .whl file from the source code in the 
+    [Py client wrapper](https://github.com/oasisprotocol/sapphire-paratime/tree/main/clients/py "Py client wrapper").
+    - Install from PyPI using ```pip install oasis-sapphire-py```.
+
+2. Install the packages in requirements.txt ```pip install -r requirements.txt```
 
 ## Setup
 
 1. If running sapphire-localnet make sure to launch the 
 [local node](https://github.com/oasisprotocol/oasis-web3-gateway/tree/main/docker).
-2. Add your deployer private key to the environment variables ```export PRIVATE_KEY=<my_private_key>```.
+2. Add your deployer private key to the environment variables 
+```export PRIVATE_KEY=<my_private_key>```.
 
 
 ## Testing
 
-Some inital unit tests are located in **./test** folder. Run ```pytest``` in the terminal. 
+Some inital unit tests are located in **./test** folder. 
+Run ```pytest``` in the terminal. 
```

Comment:
Does this require a working Localnet?

Reply:
Yes, e2e tests use sapphire-localnet by default. Updated README testing guide with additional instructions.